tests/sweep_runner/test_param_sweep_runner.py

```diff
@@ -115,20 +120,90 @@ def tests_prepare_raises_error_if_too_many_sino_slices(mocker: MockerFixture):
         chunk_shape=GLOBAL_SHAPE,
         block_start=0,
     )
-    loader = make_test_loader(mocker, block=block)
-    m1 = make_test_method(mocker)
-    sweep_wrapper = make_test_method(mocker, param_1=(0,))
-    pipeline = Pipeline(loader=loader, methods=[m1, sweep_wrapper])
-    runner = ParamSweepRunner(pipeline, MPI.COMM_WORLD)
+    preview = PreviewConfig(
+        angles=PreviewDimConfig(start=0, stop=180),
+        detector_y=PreviewDimConfig(start=0, stop=100),
+        detector_x=PreviewDimConfig(start=0, stop=160),
+    )
```

Comment:
Could you reuse the `GLOBAL_SHAPE` or `PREVIEWED_SLICES_SHAPE` variables instead of the hardcoded numbers, for the sake of not having hardcoded numbers floating around where they're already defined in variables:
```suggestion
    preview = PreviewConfig(
        angles=PreviewDimConfig(start=0, stop=PREVIEWED_SLICES_SHAPE[0]),
        detector_y=PreviewDimConfig(start=0, stop=PREVIEWED_SLICES_SHAPE[1]),
        detector_x=PreviewDimConfig(start=0, stop=PREVIEWED_SLICES_SHAPE[2]),
    )
```